go build fails with 'undefined: tls.CipherSuites'

go build fails with 
```
[ansible@awx2 webhook]$ go build github.com/adnanh/webhook
go: downloading github.com/gorilla/mux v1.7.3
go: downloading github.com/go-chi/chi v4.0.2+incompatible
go: downloading gopkg.in/fsnotify.v1 v1.4.2
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/gofrs/uuid v3.2.0+incompatible
go: downloading github.com/clbanning/mxj v1.8.4
go: downloading github.com/ghodss/yaml v1.0.0
go: extracting github.com/dustin/go-humanize v1.0.0
go: extracting github.com/gofrs/uuid v3.2.0+incompatible
go: extracting gopkg.in/fsnotify.v1 v1.4.2
go: extracting github.com/gorilla/mux v1.7.3
go: extracting github.com/ghodss/yaml v1.0.0
go: downloading gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
go: downloading golang.org/x/sys v0.0.0-20191228213918-04cbcbbfeed8
go: extracting github.com/go-chi/chi v4.0.2+incompatible
go: extracting gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
go: extracting github.com/clbanning/mxj v1.8.4
go: extracting golang.org/x/sys v0.0.0-20191228213918-04cbcbbfeed8
go: finding github.com/go-chi/chi v4.0.2+incompatible
go: finding github.com/clbanning/mxj v1.8.4
go: finding github.com/gorilla/mux v1.7.3
go: finding github.com/dustin/go-humanize v1.0.0
go: finding gopkg.in/fsnotify.v1 v1.4.2
go: finding github.com/gofrs/uuid v3.2.0+incompatible
go: finding github.com/ghodss/yaml v1.0.0
go: finding gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
go: finding golang.org/x/sys v0.0.0-20191228213918-04cbcbbfeed8
# github.com/adnanh/webhook
./tls.go:11:20: undefined: tls.CipherSuites
./tls.go:53:15: undefined: tls.CipherSuites
```
Version is:
```
go version go1.13.11 linux/amd64
```
Any ideas ?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.13.x
+  - 1.14.x
   - master
 
 os:

--- a/README.md
+++ b/README.md
@@ -26,7 +26,7 @@ If you don't have time to waste configuring, hosting, debugging and maintaining 
 # Getting started
 ## Installation
 ### Building from source
-To get started, first make sure you've properly set up your [Go](http://golang.org/doc/install) 1.12 or newer environment and then run
+To get started, first make sure you've properly set up your [Go](http://golang.org/doc/install) 1.14 or newer environment and then run
 ```bash
 $ go build github.com/adnanh/webhook
 ```

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/adnanh/webhook
 
-go 1.13
+go 1.14
 
 require (
 	github.com/clbanning/mxj v1.8.4

